packages/storybook/stories/IdentityManagement/IdentityManagement.stories.tsx

```diff
@@ -0,0 +1,300 @@
+const DisplayNamesInput = (props: DisplayNamesInputProps): JSX.Element => {
+  const { displayNames, setDisplayNames } = props;
+
+  const onTextChange = useCallback(
+    (i: number, value: string | undefined) => {
+      // TODO: `draft` is unnecessary.
+      const draft = [...displayNames];
+      draft[i] = value ?? '';
+      setDisplayNames(draft);
+    },
+    [displayNames, setDisplayNames]
+  );
+  const onAddParticipant = useCallback(() => {
+    const draft = [...displayNames];
+    draft.push('');
+    setDisplayNames(draft);
+  }, [displayNames, setDisplayNames]);
+  const onRemoveParticipant = useCallback(() => {
+    const draft = [...displayNames];
+    draft.splice(draft.length - 1, 1);
+    setDisplayNames(draft);
+  }, [displayNames, setDisplayNames]);
+
+  return (
+    <Stack
+      tokens={nestedStackTokens}
+      className={mergeStyles({
+        width: '100%'
+      })}
+    >
+      {displayNames.map((name, i) => (
+        <Stack.Item key={`displayNameInput.${i}`}>
+          <TextField
+            className={mergeStyles({
+              width: '30%',
+              maxWidth: 300
+            })}
+            placeholder="Enter display name"
+            value={displayNames[i]}
+            onChange={(_, value) => onTextChange(i, value)}
+          />
+        </Stack.Item>
+      ))}
+      <Stack.Item>
+        <Stack
+          horizontal
+          tokens={nestedStackTokens}
+          className={mergeStyles({
+            width: '100%'
+          })}
+        >
+          <Stack.Item>
+            <IconButton
+              iconProps={addIcon}
+              title="Add Participant"
+              ariaLabel="Add Participant"
+              onClick={onAddParticipant}
+            />
+          </Stack.Item>
+          {displayNames.length > 1 ? (
+            <Stack.Item>
+              <IconButton
+                iconProps={removeIcon}
+                title="Remove Participant"
+                ariaLabel="Remove Participant"
+                onClick={onRemoveParticipant}
+              />
+            </Stack.Item>
+          ) : (
+            <></>
+          )}
+        </Stack>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const CopyableResponses = (props: { responses: unknown[] | undefined }): JSX.Element => {
+  const { responses } = props;
+  if (responses === undefined) {
+    return <CopyableResponse response={undefined} />;
+  }
+  return (
+    <Stack
+      tokens={nestedStackTokens}
+      className={mergeStyles({
+        width: '100%'
+      })}
+    >
+      {responses.map((response, i) => (
+        <Stack.Item key={`response.${i}`}>
+          <CopyableResponse response={response} />
+        </Stack.Item>
+      ))}
+    </Stack>
+  );
+};
+
+const CopyableResponse = (props: { response: unknown | undefined }): JSX.Element => {
+  const { response } = props;
+  const theme = useTheme();
+
+  if (response === undefined) {
+    return (
+      <Text>
+        Tip: Connection string can be{' '}
```

Comment:
personal preference nit: I'd remove the 'Tip' and just have the Connection string can b...
```suggestion
        Connection string can be{' '}
```

Reply:
Done.